### PR TITLE
Apptrack 1010 b (jk)

### PR DESCRIPTION
**Pull request by jaykurani**

### CODE REVIEW

File: src/containers/CreateVacancy/Forms/BasicInfo/BasicInfo.js
Comment (HarvBCoding) [inline src/containers/CreateVacancy/Forms/BasicInfo/BasicInfo.js:1]: Changes to this file are not necessary as committee members do not have access to this page

File: src/containers/CreateVacancy/EditVacancy.js
Comment (HarvBCoding) [inline src/containers/CreateVacancy/EditVacancy.js:1]: Changes to this file are not necessary as committee members do not have access to this page

File: src/containers/CreateVacancy/CreateVacancy.js
Comment (HarvBCoding) [inline src/containers/CreateVacancy/CreateVacancy.js:1]: Changes to this file are not necessary as committee members do not have access to this page

File: webpack/servicenow.config.js
Comment (HarvBCoding) [inline webpack/servicenow.config.js:1]: Please delete user information from repo

**Review by HarvBCoding — CHANGES_REQUESTED** [review]

**Comment by jaykurani** [comment]
Completed with another branch

**Pull request closed without merging**